roles/ovirt-host-deploy-libvirt-guests/README.md

```diff
@@ -0,0 +1,46 @@
+oVirt host deploy - libvirt-guests
+=============================
+
+This role is used for configuration of libvirt-guests on oVirt hypervisor.
+To make the libvirt-guests play nicely with the VDSM, these options are set:
+  - libvirt-guests service is started and enables
+  - optionaly set shutdown timeout
+  - optionaly enable parallel shutdown
+
+Requirements
+------------
+
+ * Ansible version 2.0
+
+Role Variables
+--------------
+
+| Name                          | Default |  Description                                       |
+|-------------------------------|---------|----------------------------------------------------|
+| host_deploy_shutdown_timeout  |     600 | Timeout before systemd starts killing processes    |
+| host_deploy_shutdown_parallel |       0 | If set VMs are shutdown in parallel with count set |
```

Comment:
Can you please document here `libvirt_guests_config` ?

Reply:
@machacekondra thank you for the input, it is done